analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -56,7 +56,7 @@ typealias ProjectResults = Map<File, List<ProjectAnalyzerResult>>
 /**
  * A class to represent a package manager with the given [managerName] that handles projects of the given [projectType].
  * The analysis of any projects and their dependencies starts in the [analysisRoot] directory using the given
- * [analyzerConfig]. Any project-specific configuration stored in the repository is passed as [repoConfig].
+ * [analyzerConfig]. Any product-specific configuration stored in the repository is passed as [repoConfig].
```

Comment:
Have you considered using the term `release artifact` or `build artifact`? (IIRC the former we already use)

Reply:
But generally I'm fine with this proposal as well. It improves IMHO.

Reply:
Not really, THB, as "product" immediately came to my mind as we had used that term in verbal communication before.

However, given that ORT server uses the term "product" [for a group of repositories](https://github.com/eclipse-apoapsis/ort-server/blob/1ecd076fbd5ebce1e1892f4df425d8d9ea9e0f14/model/src/commonMain/kotlin/Product.kt#L22-L25), we should probably indeed come up with something else.

But I believe anything with "artifact" also doesn't nail it, because you usually have one artifact per project published.

I'm now thinking of "distribution-specific".